protocol/synthetix/subgraph/matchstick.yaml

```diff
@@ -1,2 +1,2 @@
 testFolder: tests/
-libsFolder: ../node_modules/
+libsFolder: ../../../node_modules/
```

Comment:
If we add this:
manifestPath: ./subgraph.base.yaml

We can change the `test` npm script to be just `graph test`